components/auth/SignupHelper.tsx

```diff
@@ -0,0 +1,218 @@
+  //             placeholder="email@address.com"
+  //             autoCapitalize={'none'}
+  //           />
+  //         </View>
+  //         <View >
+  //           <Input
+  //             label="Password"
+  //             leftIcon={{ type: 'font-awesome', name: 'lock' }}
+  //             onChangeText={(text: any) => setPassword(text)}
+  //             value={password}
+  //             secureTextEntry={true}
+  //             placeholder="Password"
+  //             autoCapitalize={'none'}
+  //           />
+  //         </View>
+  //         <Text style={[styles.Text22]}>Password must be at least 8 characters and contain a letter and a number.</Text>
+  //         <View style={{ flexDirection: 'row', alignItems: 'center' }}>
+  //           <Text>By clicking Sign Up, you acknowledge that you have read the Privacy Policy and agree to the Terms of Service</Text>
+  //         </View>
+  //       </View>
+  //     </View>
+      
+  //     <View style={styles.Container3}>
+  //       <Button
+  //         title="Sign up"
+  //         disabled={loading}
+  //         onPress={() => signupWithEmail(email, password)}
+  //       />
+  //     </View>
+  //   </View>
+  // );
+  return (
+    <View style={[styles.container]}>
+      <View style={[styles.container1]}>
+        <Image
+          style={[]}
+          resizeMode="contain"
+          source={require('../../assets/images/Signup/Signup-Logo1.png')}
+        />
+      </View>
+
+      <View style={[styles.container2]}> 
+        <Text style={[styles.textCreate]}>Create an Account</Text>
+        <View style={[]}>
+          <View >
+            <Input
+              leftIcon={{ type: 'font-awesome', name: 'envelope' }}
+              onChangeText={(text: any) => setEmail(text)}
+              value={email}
+              placeholder="email@address.com"
+              autoCapitalize={'none'}
+            />
+          </View>
+          <View >
+            <Input
+              leftIcon={{ type: 'font-awesome', name: 'lock' }}
+              onChangeText={(text: any) => setPassword(text)}
+              value={password}
+              secureTextEntry={true}
+              placeholder="Password"
+              autoCapitalize={'none'}
+            />
+          </View>
+          <Text style={[styles.textSmall]}>Password must be at least 8 characters and contain a letter and a number.</Text>
+          <View style={[]}>
+            <Text style={[styles.textSmall]}>By clicking Sign Up, you acknowledge that you have read the Privacy Policy and agree to the Terms of Service</Text>
+          </View>
+        </View>
+      </View>
+      
+      <View style={[styles.container3]}>
+        <View style={[styles.buttonContainer1]}>
+          <SignupButton email={email} password={password} loading={loading}/>
+        </View>
+        <View style={[styles.buttonContainer2]}>
+          <Text style={[styles.textSmall]}>-------------------- or sign up with --------------------</Text>
```

Comment:
You could use a text divider here instead of typing in manually

Reply:
This is not plausible because we have no react-native-elements library.